app/views/admin/policy_groups/new.html.erb

```diff
@@ -1,13 +1,16 @@
-<% page_title "New group" %>
+<% content_for :page_title, "New group" %>
+<% content_for :title, "New group" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @policy_group)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Do not create groups without consulting GDS. Changes to groups appear instantly on the live site."
```

Comment:
I combined the two warning messages here which might be wrong @nnagewad. Adding two warnings felt a bit weird though. Let me know if you'd like it changed

Reply:
Just saw that message. As per our slack conversation, there is a lot to explore. Do we need to incorporate the second sentence at all?

For now it's good as is, but we can explore whether it's even necessary...